shell/components/ResourceList/index.vue

```diff
@@ -148,14 +153,29 @@ export default {
      * This covers case 1
      */
     namespaceFilter(neu, old) {
-      if (sameContents(neu, old)) {
-        return;
+      if (neu && !this.componentWillFetch) {
```

Comment:
Previously it looks like if the user set namespaceFilter to something falsy like null (would this be null for All Namespaces?) it would trigger a fetch. Are we intentionally ignoring this case?

Reply:
This property relates to the `Require Namespace / Project Filtering` feature. In that world we won't show the list if the user hasn't selected any namespaces or projects, so won't need to kick off a request. So this should be a safe change. Just in case i gave it a test going from config map list with `All Namespaces` selected --> enable feature --> config list and and no list was shown --> select a project --> list shown with correct rows